scripts/util-minikube.sh

```diff
@@ -0,0 +1,320 @@
+#!/bin/bash
+
+# Helper functions for minikube deployment of kubeflow
+# Linux or Mac OS X
+
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+YELLOW='\033[0;33m'
+NC='\033[0m'
+
+OSX="Darwin"
+LINUX="Linux"
+
+HOST_PLATFORM=`uname`
+if [[ ${HOST_PLATFORM} == $LINUX ]]
+then
+  fpn=`uname -a`
+  if [[ $fpn =~ *(Ubuntu|Debian)* ]]
+  then
+    DIST_TYPE="Ubuntu"
+  elif [[ $fpn =~ *(CentOS|Fedora|Red Hat)* ]]
+  then
+    DIST_TYPE="CentOS"
+  fi
+fi
+
+MINIKUBE_CMD="minikube start"
+MOUNT_LOCAL=false
+
+GB1=$(( 1024 * 1024 * 1024 ))
+MB1=$(( 1024 * 1024 ))
+KB1=$(( 1024 ))
+
+function install_ks_kubectl_minikube() {
```

Comment:
Should we make this a separate script?

Reply:
Let's discuss how to structure it. I am really looking for a single entry point script that does all of the needed installs etc. which is what the original setup-minikube.sh was. It looks like we need to build out two different experiences: 
1. straightforward one script does it all
2. With all the dev setup similar to GKE, ie. kubeflow repo etc. being present, kfctl to deploy on minikube with its split phases.

If so, I'd need to do another refactoring, moving around bunch of this stuff.

My primary goal is to get #1 ready as described in the local CUJ